cst.yml

```
@@ -8,6 +8,17 @@ fileExistenceTests:
   - name: 'Jenkins agent'
     path: '/usr/local/bin/jenkins-agent'
     shouldExist: true
+    isExecutableBy: 'any'
   - name: 'Jenkins agent jar file'
     path: '/usr/share/jenkins/agent.jar'
-    shouldExist: true
\ No newline at end of file
+    shouldExist: true
+  - name: 'directory for jenkins user created'
+    path: '/home/jenkins'
+    shouldExist: true
+    isExecutableBy: 'any'
```

```suggestion
    permissions: 'drwxr-x---' # 750
```
The "execution" right for a directory (the `x`) means that the directory can be traversed by the entity with the execution right.

For instance if you `chmod 755 /home/jenkins` meaning to apply the right `rwxr-xr-x`, then anyone could browse to this directory and list its content.

This is not a behavior we want for a home directory: only the owner and the main group should be allowed to access it, meaning having a mode of `750` (forbidding anythong from "others", read and traverse only for "group" and all rights to the owner).